Dalamud/Game/SigScanner.cs

```diff
@@ -463,6 +462,8 @@ private static int[] BuildBadCharTable(byte[] needle, bool[] mask)
 
     private void SetupSearchSpace(ProcessModule module)
     {
+        var fileBytes = File.ReadAllBytes(module.FileName);
```

Comment:
This will _always_ read all file bytes into memory, no?

If so (and we're committing to always reading `.text` from disk), is there any downside to reading *everything* from disk instead? cc @Soreepeong.

Reply:
It works better because it almost rarely is intentional to sigscan over thunk-overwritten memory data, and as long as offsets match, even using different hooking libraries to nest hooks should work fine.

Though, since it will read the file every time a class instance is constructed, probably better to do that once and share the read-only memory across different instances.

Also, we probably have IMAGE_DOS_HEADER and IMAGE_NT_HEADERS64 importable, so it might be better to use those structs instead of Marshal.Read... with byte offsets.